apis/v1alpha1/shared_types.go

```diff
@@ -175,6 +189,33 @@ type AdminNetworkPolicyEgressPeer struct {
 	// <network-policy-api:experimental>
 	// +optional
 	Nodes *metav1.LabelSelector `json:"nodes,omitempty"`
+	// Networks defines a way to select peers via CIDR blocks (both v4 & v6).
```

Comment:
I feel like k8s has reached a point where saying "both v4 and v6" is an anti-pattern. These days, everything networking-related should be assumed to apply to both IPv4 and IPv6 unless it explicitly says it _doesn't_

Reply:
ack agreed, let me fix this up